### PR TITLE
Added 'no bounties found' message

### LINKED ISSUE #60: [Feature] Add "no records found" if there are no records
norecordfoundmessagenotfound

If there are no accepted or rejected PRs, then show a suitable message like "No Records Found!!"

In reference to issue #49 .

### PR DESCRIPTION
**Pull request by VibhorCodecianGupta**
Fixes #60

### CODE REVIEW

File: views/pages/claims/view.hbs [hunk 1]
```diff
@@ -114,6 +115,15 @@
             </div>
         </div>
     {{/each}}
+   {{else}}
+      {{#if menuH.claimed}}
```
Comment (championswimmer) [inline views/pages/claims/view.hbs]: isn't the if case reversed ?

Comment (VibhorCodecianGupta) [reply views/pages/claims/view.hbs]: Could you please elaborate? I didn't quite get you. By the way, this was working as expected on my machine.

Comment (VibhorCodecianGupta) [reply views/pages/claims/view.hbs]: The first if checks for a claim, and if it isn't there, the else gets hit which again has nested ifs checking for which tab is open (claimed, accepted, rejected) and accordingly gives a response.
Could you please elaborate what seems to be the problem? I didn't quite get you. By the way, this was working as expected on my machine.

**Comment by championswimmer** [comment]
will accept i conflicts resolved

**Pull request merged**